src/js/common/containers/DowntimeNotification.jsx

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import objectValues from 'lodash/fp/values';
+import { connect } from 'react-redux';
+import { getScheduledDowntime } from '../actions';
+import Modal from '../components/Modal';
+import LoadingIndicator from '../components/LoadingIndicator';
+
+export const services = {
+  appeals: 'appeals',
+  arcgis: 'arcgis',
+  dslogon: 'dslogon',
+  emis: 'emis',
+  es: 'es',
+  evss: 'evss',
+  idme: 'idme',
+  mvi: 'mvi',
+  mhv: 'mhv',
+  tims: 'tims',
```

Comment:
I doubt this one will ever be used. For edu, we submit over SFTP and going down doesn't impact users submitting applications.